box/box-ui-elements
# fix(dropdown): Single select field hover scroll glitch

Currently there is an issue with hovering over SingleSelectField and SelectorDropdown components. The page auto scrolls whenever we hover over an item which is not the effect that we would like.

The fix -> when an item is active, we traverse up to the dom tree of that item looking for a parent class that we do want to scroll. If that element is contained in a modal, then we want to scroll the modal, and not the element itself.

Acceptance Criteria:
When a user hovers over either a SingleSelectField or an item in a SelectorDropdown
Then the page should not suddenly scroll

**To Verify this fix:**

1. SingleSelectField Flow:
Given the user is on the UI Elements styleguide
When the user goes to the Query Bar section
When the user clicks on the Modify Filters button
When the user clicks on the first field dropdown
When the user hovers over any of the items in the dropdown
Then the user should see the that page does not suddenly scroll

2. SelectorDropdown Flow:
Given the user is on the UI Elements styleguide
When the user goes to the MetadataInstanceEditor section
When the user clicks on the Add button
When the user hovers over any of the items in the dropdown
Then the user should see that the page does not suddenly scroll
